css/common.css

```diff
@@ -0,0 +1,75 @@
+.footer{
+	width: 100%;
+	
+	line-height: 100px;
```

Comment:
There are [multiple options](https://stackoverflow.com/questions/8865458/how-do-i-vertically-center-text-with-css), depending on what exactly you are trying to pull off. I'm not saying your approach is wrong, I'm just forcing you to consider potential edge cases, so that you can make a more informed, and thus hopefully better decision. ;)